docs/tasks/run_jobs.md

```diff
@@ -19,15 +19,13 @@ Run the following command to list the Queues available in your namespace.
 
 ```shell
 kubectl -n default get localqueues
-# Or use the 'queues' alias.
-kubectl -n default get queues
```

Comment:
I think we can keep this. It's neat from an end-user perspective

Reply:
Same case as in here
https://github.com/kubernetes-sigs/kueue/pull/548#discussion_r1099080107